.goreleaser.yaml

```diff
@@ -27,6 +27,7 @@ before:
 
 release:
   draft: true
+  name_template: apache-answer-{{ .Version }}-incubating-src
```

Comment:
According to https://goreleaser.com/customization/archive/, this cause the extracted folder named as "apache-answer-x.y.x-incubating-src"